sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -662,29 +666,34 @@ static struct snd_soc_dai_link *sof_card_dai_links_create(struct device *dev,
 		if (!idisp_components)
 			goto devm_err;
 	}
-	for (i = 1; i <= hdmi_num; i++) {
+	for (i = 0; i < hdmi_num; i++) {
```

Comment:
Should we separate the change of `i` to a different commit?

Reply:
Ack, it's more clear that way. Just thought i - 1 makes the line too long :)